frontend/src/components/pages/CampCreation/index.tsx

```diff
@@ -532,7 +532,13 @@ const CampCreationPage = (): React.ReactElement => {
         isScheduleSessionsFilled={isScheduleSessionsFilled}
         isRegistrationFormFilled={isRegistrationFormFilled}
       />
-      <Box w="100%" flex="1" mt={[0, "0 !important"]} pb={20}>
+      <Box
+        w="100%"
+        flex="1"
+        mt={[0, "0 !important"]}
+        pb={20}
+        overflowX="hidden"
```

Comment:
Not related, but this disables the horizontal scroll that appears in camp creation.